### PR TITLE
#163570814 Integrate office routes with POSTGRES database and authentication

### PR DESCRIPTION
**Pull request by femitj**
**What does this PR do?**

 - Updates api endpoint route that creates political office with

   - POSTGRES database
   - authentication

**Description of the task to be completed**

 - Admin creates office and admin alone can gets ALL and gets a specific record

Ensure the following endpoint are working as they should

 - POST/api/v1/offices
 - GET/api/v1/offices
 - GET/api/v1/offices/:id

 - Implement token based authentication
   - using JSON web token
 - secure route

How should this be tested manually
 - After you clone the repo,
 - cd into it and run npm install to install node modules.
 - Make sure you have nodejs and express installed.
 - Afterwards, run npm start to initiate the app.
 - test endpoint above using postman with localhost:8080 as the host

**What are the relevant pivotal stories**
#163570814

### CODE REVIEW

File: src/controllers/OfficeControllers.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      req.body.createdOn,
+    ];
+ 
+    try {
+      const { rows } = await db.query(createQuery, values);
+      return res.status(201).send({
+        status: 201,
+        data: [{
+          id: rows[0].id,
+          message: 'Political office created',
```
Comment (andela-gike) [inline src/controllers/OfficeControllers.js]: Same here the challenge spec says you should include office name and type

File: src/models/database.js [hunk 2]
```diff
@@ -35,7 +36,33 @@ const createUserTable = () => {
       console.log(err);
```
Comment (andela-gike) [inline src/models/database.js]: Remove the console.log statement

**Review by andela-gike — CHANGES_REQUESTED** [review]

File: src/controllers/OfficeControllers.js (hunk 1 above)
Comment (femitj) [reply src/controllers/OfficeControllers.js]: done

File: src/models/database.js (hunk 2 above)
Comment (femitj) [reply src/models/database.js]: done

**Review by andela-gike — APPROVED** [review]

**Pull request merged**